rita_bin/src/database_migration.rs

```diff
@@ -38,6 +38,13 @@ fn main() {
 
     let system = actix_async::System::new();
 
+    system.block_on(db_migration_user_admin(
+        web3_url.clone(),
+        private_key,
+        private_key.to_address(),
```

Comment:
using actix async main (examples in the other bins) you can avoid the block on.

Reply:
done